web-client/iron-remote-desktop/public/package.json

```diff
@@ -10,7 +10,7 @@
     "Alexandr Yusuk"
   ],
   "description": "Backend-agnostic Web Component for remote desktop protocols",
-  "version": "0.5.1",
+  "version": "0.5.2",
```

Comment:
note: I prefer having a separate PR `chore(release): prepare iron-remote-desktop vX.Y.Z` so it’s easier to discover the bumps